### PR TITLE
Update `hardhat account`, support solana

### PR DESCRIPTION
**Pull request by lukema95**
related issue: #187 
1. Support automatic generation of Solana private keys. When using the `--save` option, store it in the `SOLANA_PRIVATE_KEY` environment variable in the .env file.
2. Support recovery using Solana private key when using the `--recover` option. When `~/.config/solana/id.json` exists locally, automatically load that private key. When it doesn't exist locally, users can input the private key in the command line or skip this step.
3. Maintain backward compatibility. Locally, store `PRIVATE_KEY`, `EVM_PRIVATE_KEY`, and `BTC_PRIVATE_KEY` environment variables in the .env file, and their values are the same.

## Summary by CodeRabbit

- **New Features**
  - Introduced support for Solana wallets alongside EVM wallets.
  - Added functionality to retrieve Solana wallets from local files or user input.
  - Enhanced console output to display Solana wallet details upon creation.

- **Improvements**
  - Updated prompts for clarity when entering recovery information.
  - Enhanced environment variable management to include Solana private keys.

### CODE REVIEW

**Comment by lukema95** [comment]
```shell
npx hardhat account --save  

    🔑 EVM Private key: 49db34db2e853bda91c17828eb614173573df9ec200bc563217aeb36687f2e0a

    🔑 Solana Private key: 2kP9uN3ySc25UNqhqqTWz5Vt1vRQpcjfgmcJ346NERKaTwUq99nwbVsPg61znaMTnwBHbYyAGnkKRmKgjJhhWq73

    🔐 EVM Mnemonic phrase: level treat gossip actress mistake parent trouble excite wild tone offer time

    😃 EVM address: 0xD7043474A099Cad2a0A3c12F746BA5A9aD74e85f
    😃 Bitcoin address: tb1qg5xxzl4qzn6pgee53a7kjve8lvewk43r8kpcjc
    😃 Bech32 address: zeta16uzrga9qn89d9g9rcyhhg6a94xkhf6zlne0vkz

    😃 Solana address: 2bZFnp4xkrhovr1ahDiuu7D8JxdoaWPyrptF4vDuy7ru
```

**Comment by lukema95** [comment]
@fadeev pls check

**Comment by fadeev** [comment]
Looking good!

File: packages/tasks/src/account.ts [hunk 1]
```diff
@@ -43,39 +46,105 @@ export const getWalletFromRecoveryInput = async (ethers: any) => {
...
+  solanaWallet &&
+    console.log(`
+    🔑 Solana Private key: ${bs58.encode(solanaWallet.secretKey)}`);
+
+  mnemonic &&
+    console.log(`
+    🔐 EVM Mnemonic phrase: ${mnemonic.phrase}`);
+
+  console.log(`
+    😃 EVM address: ${address}
```
Comment (fadeev) [inline packages/tasks/src/account.ts:136]:
```suggestion
😃 EVM address: ${address}
```

Please, remove whitespace in the beginning of the line.

**Review by andresaiello — APPROVED** [review]

**Pull request merged**